runner/executer.go

```diff
@@ -0,0 +1,67 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/cybozu-go/github-actions-controller/metrics"
+)
+
+type Executer interface {
```

Comment:
I think `Executor` is correct.

Reply:
Why do you export Executor?
Executor is only used in runner package.

Reply:
Thanks for the review.
I misunderstood the need for regret when making a mock for testing.
I have made private some related things that do not need to be exposed.